sdk/storage/storage-blob/test/blockblobclient.spec.ts

```diff
@@ -126,9 +132,9 @@ describe("BlockBlobClient", () => {
 
     // When testing is in Node.js environment with shared key, setAccessPolicy will work
     // But in browsers testing with SAS tokens, below will throw an exception, ignore it
```

Comment:
Why not try to set the access policy only after checking if we are testing in Node.js env?